src/components/AccountLinks/AccountLinksTs.ts

```diff
@@ -39,7 +39,7 @@ export class AccountLinksTs extends Vue {
   /// end-region computed properties getter/setter
 
   public get explorerUrl() {
-    return this.explorerBaseUrl + '/account/' + this.account.address
+    return this.explorerBaseUrl + 'account/' + this.account.address
```

Comment:
this implies that the configured explorerBaseUrl comes with an ending `/`.

Please use something like: `return this.explorerBaseUrl.replace(/\/+$/,'') + '/account';`

as the above will permit configuration values to hold with or without slash.